packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -277,6 +277,20 @@ pc.setRemoteSessionDetails = (peerConnection, typeStr, remoteSdp, meetingId) =>
       })
       .catch((error) =>  {
         LoggerProxy.logger.error(`Peer-connection-manager:index#setRemoteDescription --> ${error} missing remotesdp`);
+
+
+        const metricName = METRICS_OPERATIONAL_MEASURES.PEERCONNCTION_FAILURE;
+        const data = {
+          correlation_id: meetingId,
+          reason: error.message,
```

Comment:
Above you had `reason: error.error?.message,`, is it the same here?

Reply:
no actually the join function creates a custom error object, so its only one function where we have error.error?.message

Reply:
Once we implement the error code for individual errors then we should not have this issues we will send error codes

Reply:
Ok